test/e2e/runner/main.go

```diff
@@ -211,7 +221,7 @@ func NewCLI() *CLI {
 		Short: "Stops the Docker testnet",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			logger.Info("Stopping testnet")
-			return execCompose(cli.testnet.Dir, "down")
+			return cli.infp.StopTestnet(context.Background())
```

Comment:
in the short description above, update to include DO?

Reply:
Removed reference to Docker